newIDE/app/src/VariablesList/VariablesList.js

```diff
@@ -1419,7 +1419,7 @@ const VariablesList = (props: Props) => {
     (newName: string, additionalContext: any) => {
       const parsedContext = JSON.parse(additionalContext);
       const nodeId: string = parsedContext.nodeId;
-      const depth: number = parsedContext.number;
+      const depth: number = parsedContext.depth;
```

Comment:
🤦‍♂️ 
Of course I had to make a mistake in the only non typed part of the editor

Reply:
duh